Cargo.toml

```diff
@@ -10,6 +10,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 futures = "0.3.25"
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Static link openssl

Reply:
I'd rather we let users choose their openssl so they can upgrade on their own, imo this is better for a lib.

Couldn't we just install ssl before in the CI ?